.gitlab-ci.yml

```
@@ -18,8 +18,7 @@ build_cuda:
         -DCMAKE_BUILD_TYPE=Release
         -DTRACCC_BUILD_TESTING=ON
         -DTRACCC_BUILD_CUDA=ON
-      - cmake --build build
-
+      - cmake --build build -- -j 2
```

Ahh, I should've caught it earlier... 😦 The best would be to use:

```suggestion
      - cmake --build build --parallel 2
```

That formalism forward the parameter correctly for any backend. (Including Xcode and MSVC.) I know that it's not really relevant in this specific build, it's more about the principle. 😛

---

OK Fixed